### PR TITLE
add Ukrainian language

### PR DESCRIPTION
**Pull request by fasero1**
Hello, I double check this pull request https://github.com/VazkiiMods/Botania/pull/4358. Make some changes, and think that uk_ua language is ready to merge. This localisation was created for Ukrainian minecraft project, and will be modify if players find some issue.

### CODE REVIEW

File: Xplat/src/main/resources/assets/botania/lang/uk_ua.json [hunk 1]
```diff
@@ -0,0 +1,3577 @@
...
+    "effect.botania.feather_feet": "Пір'яні ноги",
+    "effect.botania.emptiness": "Порожнеча",
+    "effect.botania.bloodthirst": "Жага крові",
+    "effect.botania.allure": "Заманювання",
+    "effect.botania.clear": "Прощення",
+  
+    "item.botania.record_gaia_1.desc": "Kain Vinosec - Витримай порожнечу",
+    "item.botania.record_gaia_2.desc": "Kain Vinosec - Борись за спокій",
+  
+    "block.minecraft.banner.botania_flower.black": "Чорна містична квітка",
```
Comment (TheRealWormbo) [inline Xplat/src/main/resources/assets/botania/lang/uk_ua.json]: All banner entry keys seem to have the dot (".") after "botania" replaced with an underscore ("_"), compared to `en_us.json`, which means they are no longer the same entry.

File: Xplat/src/main/resources/assets/botania/lang/uk_ua.json [hunk 2]
```diff
@@ -0,0 +1,3577 @@
...
+    "botania.page.gIntro2": "Ці квіти прив’язуються до одного $(l:mana/spreader)$(item)Розсіювач мани$(0)$(/l) одночасно для виведення $(thing)Мани$(0). Вони прив’яжуться до найближчого розсіювача, наявного під час їх розміщення (або найближчого розміщеного пізніше, якщо жоден не був у радіусі), і їх можна прив’язати до іншого розсіювача за допомогою жезла в $(thing)Режимі прив'язки$(0), за бажанням.",
+    "botania.page.gIntro3": "Вибір вашої першої $(thing)Генерувальної флори$(0) є досить простим завданням; найпростішими двома варіантами для будь-якого ботаніка-початківця є $(l:generating_flowers/endoflame)$(item)Жаржина$(0)$(/l) і $(l:generating_flowers/hydroangeas)$(item)Гідрогортензія$(0)$(/l).$(p)Перший використовує $(item)Вугілля$(0) або інші джерела палива для створення $(thing)Мани$(0). Останній використовує джерела $(item)Води$(0) для того самого завдання, але в'яне приблизно через три дні.",
+    "botania.page.gIntro4": "Later on, you may wish to branch out to other $(thing)Generating Flora$(0). Flowers that require more complex infrastructure to automate typically produce more $(thing)Mana$(0).",
```
Comment (TheRealWormbo) [inline Xplat/src/main/resources/assets/botania/lang/uk_ua.json]: This entry is copied without translation and is not just one of those quips under a recipe.

**Review by TheRealWormbo — CHANGES_REQUESTED** [review]
I noticed some smaller things, as pointed out in the review commends above.
Also, some of the book entries (botania.entry.flowers, and .marimorphosis) have their page translations ordered differently, same for some of the subtitle translations (spreader/prism lenses). That's not a functional issue, it just shows up when comparing `uk_ua.json` to `en_us.json` and might be a bit confusing for later updates. (Translations for individual entries are mostly ordered as they appear in the book, for consistency.)

The translation keys for the banner patterns will need to be adjusted to actually be recognized by the game:
Screenshot of the tooltip for a white banner with black potato face without translation of the latter

**Review by TheRealWormbo — APPROVED** [review]

**Comment by fasero1** [comment]
Thank you for reviewing my pull request in detail.

**Comment by TheRealWormbo** [comment]
Upon further inspection, I think we both missed the underscores in the other banner pattern entries. `en_us.json` has no instance of the string "botania_" anywhere, so that will need to be fixed, but that should be it.

**Pull request merged**